lib/surface_bulma/dropdown/trigger.ex

```diff
@@ -0,0 +1,21 @@
+defmodule SurfaceBulma.Dropdown.Trigger do
+  use Surface.Component
+
+  @moduledoc """
+  Trigger for opening the dropdown menu
+  """
+
+
+  def render(assigns) do
+    ~F"""
+    <div class="dropdown-trigger">
+      <button class="button" aria-haspopup="true" aria-controls="dropdown-menu">
+        <span>Dropdown button</span>
+        <span class="icon is-small">
```

Comment:
`<Icon.FontAwesome />` does not support aria-label yet, so I guess that is why it is not used here?

Reply:
This mirrors the mark-up used in the Bulma documentation. I took for granted that they were using the most appropriate ARIA-attribute structure. I suspect the reasoning is that `aria-controls` includes the necessary semantic information and therefore labelling the  decorative icon-element is unnecessary.

Reply:
Sorry to cut the conversation short, but I guess this file is duplicate, since the markup also exists in the Dropdown component.

Reply:
Yes I see that now. I must have decided to collapse it but forgot to delete the extra file.